rest/doc_api.go

```diff
@@ -370,14 +370,25 @@ func (h *handler) handlePutDocReplicator2(docid string, roundTrip bool) (err err
 	if err != nil {
 		return err
 	}
-	if bodyBytes == nil || len(bodyBytes) == 0 {
-		return base.ErrEmptyDocument
-	}
 
-	newDoc := &db.Document{
-		ID: docid,
+	hasAttachments := bytes.Contains(bodyBytes, []byte(db.BodyAttachments))
+	hasExpiry := bytes.Contains(bodyBytes, []byte(db.BodyExpiry))
+
+	var body db.Body
```

Comment:
again, not used... not sure why it's scoped up here

Reply:
Will go back to this once handleRev is done....